src/Stratis.Bitcoin.Tests/Utilities/AsyncManualResetEventTest.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Tests.Utilities
+{
+    public class AsyncManualResetEventTest
+    {
+        /// <summary>
+        /// Tests that:
+        /// <list type="bullet">
+        /// <item><see cref="AsyncManualResetEvent"/> does not crush.</item>
```

Comment:
well :-) that was kind of a joke to put in the contrast what it really tests and what it does not test
to point out that maybe it was intentional to actually test what it did not test ...

Reply:
Kk, I'll remove the comment.